addons/event/static/src/client_action/event_barcode.scss

```diff
@@ -24,6 +24,35 @@
         max-width: 200px;
         max-height: 100px;
     }
+    .o_barcode_mobile_container {
+        margin-top: 75px;
+        margin-bottom: -35px;
+
+        img {
+            height: 185px;
+            width: 275px;
+        }
+        .o_mobile_barcode {
+            opacity: 0;
```

Comment:
opacity-0 in the template?

Reply:
Yes. This is the part that makes the barcode image clickable. However, since we don't need it visible in the view, we're setting its opacity to 0.